### ISSUE TITLE
Win11 + Radeon 780M + transparency/opacity not working

### ISSUE THREAD

**Issue by wu0lss4j** [alacritty/alacritty#8017]
Running Win11 on a AMD Radeon 780M. I use alacritty mostly with WSL2 running Debian. Basically, transparency doesn't work at any opacity level. It will only make the background color less opaque, but not see through. I.e.: if I set the primary.background to yellow, I will get a hideous yellow brackground at opacity 1.0 and 50% yellow 50% black at opacity 0.5 and 100% black at opacity 0.0, meaning that the opacity is affecting the background color but not the seethroughness of the app.

I believe this maybe is related to the graphics card and in reddit some people mentioned disabling PSR → https://www.reddit.com/r/thinkpad/comments/18k519l/stuttering_on_new_t14s_gen4_amd_mouse_cursor/ ← caveat emptor as people complained this made their computer run hot.

What I don't understand is, windows terminal has a translucent effect which works. Why doesn't Alacritty's?

My .toml config as follows:

```
import = ["C:/Users/X/AppData/Roaming/alacritty/themes/themes/rose-pine.toml"]

[env]
TERM = "xterm-256color"

[shell]
program = "wsl.exe"
args = ["-d Debbie -u X --cd ~"]

[scrolling]
history = 10000

[window]
position = { x = 0, y = 0 }
#dimensions = { columns = 160, lines = 80 }
padding = { x = 4, y = 4}
dynamic_padding = false
decorations = "full"
opacity = 0.7
# MacOS/Wayland only...
#blur = true 
startup_mode = "Maximized"
dynamic_title = true

# When true, bold text is drawn using the bright variant of colors. USELESS AND NOT WORKING
#draw_bold_text_with_bright_colors = true

#[colors]
#[colors.primary]
#background = "#000000"

[font]
size = 12
offset = { x = 0, y = 0 }
glyph_offset = { x = 0, y = 0 }
[font.normal]
family = "VictorMono NFM"
style = "Regular"

#[font.bold]
#family = "Hack Nerd Font Mono"
#style = "Bold"

#[font.italic]
#family = "MesloLGS Nerd Font Mono"
#style = "Italic"

[bell]
animation = "EaseOutExpo"
duration = 0

[mouse]
#double_click.threshold = 300 # not working
#triple_click.threshold = 300 # not working
hide_when_typing = true
#faux_scrolling_lines = 1 # not working

[selection]
semantic_escape_chars = ",│`|:\"' ()[]{}<>"
save_to_clipboard = true

[cursor]
style.shape = "Beam"
style.blinking = "On"
# VI mode USELESS AND NOT WORKING
#vi_mode_style.style = "Block"
#vi_mode_style.blinking = "Off"
```

### System

OS: Microsoft Windows [Version 10.0.22631.3672]
Version: 0.13.2
Linux/BSD: Debian running inside WSL2

### Logs

Crashes: no crashes...

Font/Terminal size:

[0.000008000s] [INFO ] [alacritty] Welcome to Alacritty
[0.000303000s] [INFO ] [alacritty] Version 0.13.2 (bb8ea18)
[0.002817600s] [INFO ] [alacritty] Configuration files loaded from:
                                     "C:\\Users\\X\\AppData\\Roaming\\alacritty\\alacritty.toml"
[0.017753000s] [INFO ] [alacritty] Window scale factor: 1
[0.152184000s] [INFO ] [alacritty] Using WGL
[0.152651800s] [DEBUG] [alacritty] Picked GL Config:
                                     buffer_type: Some(Rgb { r_size: 8, g_size: 8, b_size: 8 })
                                     alpha_size: 8
                                     num_samples: 0
                                     hardware_accelerated: true
                                     supports_transparency: Some(false)
                                     config_api: Api(OPENGL | GLES1 | GLES2)
                                     srgb_capable: true
[0.170066300s] [DEBUG] [alacritty] Loading "Consolas" font
[0.242809100s] [INFO ] [alacritty] Running on AMD Radeon(TM) 780M
[0.242950100s] [INFO ] [alacritty] OpenGL version 3.3.0 Core Profile Context 24.5.1.240502, shader_version 4.60
[0.242996600s] [INFO ] [alacritty] Using OpenGL 3.3 renderer
[0.275313100s] [DEBUG] [alacritty] Enabled debug logging for OpenGL
[0.275504800s] [DEBUG] [alacritty] Filling glyph cache with common glyphs
[0.276296000s] [INFO ] [alacritty] Cell size: 8 x 17
[0.276367600s] [INFO ] [alacritty] Padding: 0 x 0
[0.276402700s] [INFO ] [alacritty] Width: 800, Height: 600
[0.329393900s] [INFO ] [alacritty] PTY dimensions: 35 x 100
[0.330460900s] [INFO ] [alacritty_terminal] Using Windows API for pseudoconsole
[0.336345100s] [INFO ] [alacritty] Initialisation complete

Keyboard and bindings:

[0.000007100s] [INFO ] [alacritty] Welcome to Alacritty
[0.000292300s] [INFO ] [alacritty] Version 0.13.2 (bb8ea18)
[0.002490300s] [INFO ] [alacritty] Configuration files loaded from:
                                     "C:\\Users\\X\\AppData\\Roaming\\alacritty\\alacritty.toml"
                                     "C:/Users/X/AppData/Roaming/alacritty/themes/themes/rose-pine.toml"
[0.003025800s] [INFO ] [alacritty] winit event: NewEvents(Init)
[0.003096900s] [INFO ] [alacritty] winit event: Resumed
[0.079783900s] [INFO ] [alacritty] Window scale factor: 1
[0.241053600s] [INFO ] [alacritty] Using WGL
[0.337675200s] [INFO ] [alacritty] Running on AMD Radeon(TM) 780M
[0.337859600s] [INFO ] [alacritty] OpenGL version 3.3.0 Core Profile Context 24.5.1.240502, shader_version 4.60
[0.337905700s] [INFO ] [alacritty] Using OpenGL 3.3 renderer
[0.419272500s] [INFO ] [alacritty] Cell size: 8 x 22
[0.419532800s] [INFO ] [alacritty] Padding: 4 x 4
[0.419594200s] [INFO ] [alacritty] Width: 1920, Height: 1009
[0.461003100s] [INFO ] [alacritty] PTY dimensions: 45 x 239
[0.462498100s] [INFO ] [alacritty_terminal] Using Windows API for pseudoconsole
[0.469511200s] [INFO ] [alacritty] Initialisation complete

**Comment by chrisduerr** [comment]
> I believe this maybe is related to the graphics card

I have no interest in chasing down graphics driver bugs. Especially on Windows.

**Comment by wu0lss4j** [comment]
For future reference I guess. 

Y-day 05.07.2024, I updated to the latest AMD Radeon drivers available but did not restart the computer upon request.

Today 06.07.2024, I fired up alacritty to check something and was surprised by the transparency → 
https://imgur.com/a/nvsaHii

I guess... my guess only... that AMD drivers interfere somehow with the rendering. I tried to run AMD Adrenalin but it won't start until I reboot and finish installing the updated drivers.

Food for thought.
